frontend/src/components/Login.js

```diff
@@ -1,16 +1,14 @@
 import React from "react";
```

Comment:
can remove this

Reply:
Remove 'import React from "react" '? Gets thrown into an error state if I do.

Reply:
scratch that!